step-sequencer/index.html

```diff
@@ -216,7 +206,7 @@ <h2>DTMF</h2>
       attackControl.addEventListener(
         "input",
         (ev) => {
-          attackTime = parseInt(ev.target.value, 10);
+          attackTime = parseFloat(ev.target.value, 10);
```

Comment:
From what I understand parseFloat takes a single argument:
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/parseFloat
```suggestion
          attackTime = parseFloat(ev.target.value);
```

Reply:
Super, copy paste err - well spotted 👍🏻